report/src/main/webapp/app/services/metadataEndpoint.service.js

```diff
@@ -15,20 +15,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-define(['angularAMD', 'configService', 'requestParametersService'], function (angularAMD) {
+define(['angularAMD', 'endpointConfiguration', 'requestParametersService'], function (angularAMD) {
 	'use strict';
 	angularAMD.factory('metadataEndpointService', MetadataEndpointService);
 
 	/**
 	 * Service responsible for communication with AET metadata REST API endpoint.
 	 */
-	function MetadataEndpointService($q, $http, configService, requestParametersService) {
+	function MetadataEndpointService($q, $http, endpointConfiguration, requestParametersService) {
 		var service = {
 				getMetadata: getMetadata,
 				saveMetadata: saveMetadata
 			},
 			requestParams = requestParametersService.get(),
-			configParams = configService.getConfig();
+			getEndpointUrl = endpointConfiguration.getEndpointUrl();
```

Comment:
Could we rename `getEndpointUrl` variable? To sth like
- `metadataUrl`
- `endpoint`
- ??